tests/end_to_end/test_emotet/data/playbooks/configuration.yaml

```diff
@@ -0,0 +1,122 @@
+- name: Test agent configuration
```

Comment:
We cannot upload an `.exe` file to the repository. This type of auxiliary files must be uploaded to S3 and downloaded in the test configuration phase.